src/editor/EditorAdditions.js

```diff
@@ -24,16 +24,19 @@ const EditorAdditions = () => {
 
 	// Render a preview of the dismiss button at the end of the block content area.
 	useEffect(() => {
+		let dismissButtonPreview = document.querySelector(
+			'.newspack-popups__not-interested-button-preview'
+		);
+
 		if ( ! dismiss_text ) {
+			if ( dismissButtonPreview ) {
+				dismissButtonPreview.remove();
```

Comment:
Note:[ `remove`](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove#browser_compatibility) will not work in IE

Reply:
Right - how's this? https://github.com/Automattic/newspack-popups/pull/402/commits/a6c2bec69171d16c855b216e2905a286f88a505b

Reply:
👌